legend-sdlc-generation-service/src/main/java/org/finos/legend/sdlc/generation/service/ServiceExecutionClassGenerator.java

```diff
@@ -85,13 +93,19 @@ public GeneratedJavaClass generate()
             this.javaImports.stream().sorted().map(i -> "import " + i + ";\n").forEach(builder::append);
         }
         builder.append('\n');
-        builder.append("public class ").append(this.service.name).append(" extends ").append(AbstractServicePlanExecutor.class.getSimpleName()).append('\n');
+        builder.append("public class ").append(this.service.name).append(" extends ").append(AbstractServicePlanExecutor.class.getSimpleName()).append(" implements ").append(ServiceRunner.class.getSimpleName()).append('\n');
```

Comment:
This was done to ensure that this is not a breaking change (unless this change goes in, generated execution classes would fail to compile due to missing run method implementation). I plan to to have AbstractServicePlanExecutor implement ServiceRunner going forward.

Reply:
It will be better to get AbstractServicePlanExecutor into the proper form before we release this to users. Is it as simple as adding a default implementation of run to AbstractServicePlanExecutor?

Reply:
Should be addressed in https://github.com/finos/legend-engine/pull/356